frontend/src/services/async-storage.service.ts

```diff
@@ -0,0 +1,29 @@
+import * as SecureStore from 'expo-secure-store'
```

Comment:
Deze file is gemaakt om het managen van de userdata over de hele applicatie makkelijker te maken. Dit is in plaats van die Cookies want met cookies zijn er best wel limitations and dingen die niet goed werken met react native. Wat we hier gebruiken is een async encrypted storage. 

Er is veel boilerplate code dus om die reden heb ik het makkelijker gemaakt.